development-kit/pkg/engines/nodejs/rules.go

```diff
@@ -16,13 +16,66 @@
 package nodejs
 
 import (
+	engine "github.com/ZupIT/horusec-engine"
 	"github.com/ZupIT/horusec-engine/text"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/nodejs/and"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/nodejs/or"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/nodejs/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/nodejs/and"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/nodejs/or"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/nodejs/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/utils/logger"
 )
 
-func AllRulesNodeJSRegular() []text.TextRule {
+type Interface interface {
+	GetAllRules() (rules []engine.Rule)
+	GetTextUnitByRulesExt(projectPath string) ([]engine.Unit, error)
+}
+
+type Rules struct{}
+
+func NewRules() Interface {
+	return &Rules{}
+}
+
+func (r *Rules) GetAllRules() (rules []engine.Rule) {
+	rules = r.addRules(rules)
+	return rules
+}
+
+func (r *Rules) addRules(rules []engine.Rule) []engine.Rule {
+	for index := range allRulesNodeJSAnd() {
+		rules = append(rules, allRulesNodeJSAnd()[index])
```

Comment:
Change for usage item. See commentary in java/rules for more details

Reply:
Fixed